code/game/objects/items/storage/boxes.dm

```diff
@@ -284,6 +284,18 @@
 		/obj/item/dnainjector/m2h = 3)
 	generate_items_inside(items_inside,src)
 
+/obj/item/storage/box/surgery
+  name = "Emergency Surgery Kit"
+  desc = "A box containing the bare necessities for healing a patients wounds."
+  illustration = "surgery"
+
+/obj/item/storage/box/surgery/PopulateContents()
+  new /obj/item/surgical_drapes(src)
+  new /obj/item/reagent_containers/medigel/sterilizine(src)

```

Comment:
remove sterilizine, it shouldn't be common especially when you're giving them the tools that give perfect probability.